src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
+Actors:
+  - Name: MultipleCollectionsPerLoaderThread
+    Type: Loader
+    Threads: 10
+    Phases:
+      # If CollectionCount is less than the PhaseConfig Threads, then unexpected outcomes will occur.
+      - Repeat: 1
+        # create 6 collection (CollectionCount / PhaseConfig Threads) per thread over 10 threads.
```

Comment:
Oh, is this because threads below is 1, each (of the 10) thread creates 6 collections? Wow.

Reply:
Yes, hence the name **MultipleCollectionsPerLoaderThread**.

(Also all the indexes will be created once all the loading is complete).